docs/install.md

```diff
@@ -60,12 +60,16 @@ Nextflow is distributed as a self-installing package, in order to make the insta
     chmod +x nextflow
     ```
 
-3. Move Nextflow into an executable path:
+3. Move Nextflow into an executable path. For example:
 
     ```bash
-    sudo mv nextflow /usr/local/bin
+    mv nextflow $HOME/.local/bin/
```

Comment:
Not sure about this, because `$HOME/.local/bin` is not a standard path. Mac does not have it (not sure across different linux distribution).

Reply:
Good point - it would be nice to have a suggested path rather than saying "Install on your path". Do you have any suggestions?

Reply:
maybe it can be fixed adding `mkdir -p $HOME/.local/bin/` before `mv` 

and changing the tip something like:

> make sure the directory `$HOME/.local/bin/` is in your `PATH` variable, otherwise .. (how to do it). Alternatively you can use any other directory already in your path. Be aware nextflow will try to override this file  when upgrading it, etc ..